common/build.gradle.kts

```diff
@@ -68,10 +69,12 @@ kotlin {
 
     sourceSets["androidMain"].dependencies {
         implementation(deps.kotlin.stdlib)
+        implementation("com.squareup.sqldelight:android-driver:1.4.0")
     }
 
     sourceSets["iosMain"].dependencies {
         implementation(deps.kotlin.xcode)
+        implementation("com.squareup.sqldelight:native-driver:1.4.0")
```

Comment:
These can also be added to the `deps` object in `buildSrc`. A good pattern would be something like `deps.sqldelight.android` and `deps.sqldelight.android`.

We also need to add the JavaScript driver to the `jsMain` source set.